src/components/Loader.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-    <div class="loader" :class="showLoadingBackground && (lightBlue ? 'nq-light-blue-bg' : 'nq-blue-bg')">
+    <div class="loader" :class="[showLoadingBackground && (lightBlue ? 'nq-light-blue-bg' : 'nq-blue-bg'), state === 'success' && 'exit-transition']">
```

Comment:
Would be nicer to use
```suggestion
    <div class="loader" :class="{ 'nq-blue-bq': showLoadingBackground && !lightBlue, 'nq-light-blue-bg': showLoadingBackground && lightBlue, 'exit-transition': state === 'success' }">
```